packages/server/src/endpoints/locations.js

```diff
@@ -11,6 +11,22 @@ const getLocations = {
   },
 };
 
+const postLocation = {
+  method: 'POST',
+  path: '/locations',
+  async handler(request, h) {
+    const { payload } = request;
```

Comment:
You don't have to change this, but if you want you could destructure `request` in the parameters.

```js
async handler({ payload }, h) {
}
```

I'm torn between what I like more. Destructuring in the params seems slicker for something like this, but can get harder to read if you have many (especially if you're renaming them).